docs/data/joy/components/grid/ChildrenCenteredGrid.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import Avatar from '@mui/joy/Avatar';
+import Box from '@mui/joy/Box';
+import Grid from '@mui/joy/Grid';
+import Typography from '@mui/joy/Typography';
+
+export default function ChildrenCenteredGrid() {
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <Grid
+        container
+        spacing={2}
+        sx={{
+          '--Grid-borderWidth': '1px',
+          borderTop: 'var(--Grid-borderWidth) solid',
+          borderLeft: 'var(--Grid-borderWidth) solid',
+          borderColor: 'divider',
+          '& > div': {
+            borderRight: 'var(--Grid-borderWidth) solid',
+            borderBottom: 'var(--Grid-borderWidth) solid',
+            borderColor: 'divider',
+          },
+        }}
+      >
+        {['Jimmy', 'Michal', 'Jun', 'Marija'].map((name, index) => (
+          <Grid
+            key={index}
+            {...{ xs: 12, sm: 6, md: 3 }}
```

Comment:
Is there any reason for using spread? I understand that it does not make a difference in this case but let's make it consistent.
```suggestion
            xs={12}
            sm={6}
            md={3}
```